src/components/widgets/spoolman/SpoolSelectionDialog.vue

```diff
@@ -1,205 +1,201 @@
-                          <b>{{ $filters.getReadableWeightString(item.remaining_weight) }}</b>
-                          / {{ $filters.getReadableWeightString(item.filament.weight) }}
-                        </small>
-                      </div>
-                    </div>
-                  </div>
-                </td>
-                <td
-                  v-for="header in visibleHeaders.filter(h => h.value !== 'filament_name')"
-                  :key="header.value"
-                >
-                  {{ item[header.value] }}
-                </td>
-              </tr>
-            </template>
-          </v-data-table>
-        </v-card-text>
-
-        <v-divider />
-
-        <v-card-actions
-          class="pt-4"
-        >
+  <app-dialog
+    v-model="open"
+    scrollable
+    :max-width="isMobileViewport ? '90vw' : '75vw'"
+    title-shadow
+  >
+    <template #title>
+      <span class="focus--text">{{ $t('app.spoolman.title.spool_selection') }}</span>
+
+      <v-spacer />
+
+      <v-menu
+        v-if="cameras.length > 1"
+        left
+        offset-y
+        transition="slide-y-transition"
+      >
+        <template #activator="{ on, attrs, value }">
           <app-btn
-            v-if="spoolmanURL"
-            :href="spoolmanURL"
-            target="_blank"
+            v-bind="attrs"
+            small
+            class="ms-1 my-1"
+            v-on="on"
           >
+            <v-icon
+              class="mr-1"
+              small
+            >
+              $camera
+            </v-icon>
+            <template v-if="!isMobileViewport">
+              {{ $t('app.spoolman.btn.scan_code') }}
+            </template>
             <v-icon
               small
-              class="mr-2"
+              class="ml-1"
+              :class="{ 'rotate-180': value }"
             >
-              $edit
+              $chevronDown
             </v-icon>
-            {{ isMobileViewport ? '' : $tc('app.spoolman.btn.manage_spools') }}
           </app-btn>
-
-          <v-spacer />
-
-          <app-btn
-            text
-            color="warning"
-            @click="open = false"
+        </template>
+        <v-list dense>
+          <v-list-item
+            v-for="camera in cameras"
+            :key="camera.id"
+            @click="scanSource = camera.id"
           >
-            {{ $t('app.general.btn.cancel') }}
-          </app-btn>
-          <app-btn
-            color="primary"
-            @click="handleSelectSpool"
+            <v-list-item-icon>
+              <v-icon>
+                $camera
+              </v-icon>
+            </v-list-item-icon>
+            <v-list-item-content>
+              <v-list-item-title>
+                {{ camera.name }}
+              </v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+        </v-list>
+      </v-menu>
+
+      <app-btn
+        v-else-if="cameras.length"
+        small
+        class="ms-1 my-1"
```

Comment:
```suggestion
            class="mr-2"
```

This feels a bit crammed, and I'm not sure if the button should be aligned with the search input box?

Before
![image](https://github.com/matmen/fluidd/assets/25269274/7ed08a6b-5529-45c4-8711-adfd32dce531)
After
![image](https://github.com/matmen/fluidd/assets/25269274/02a8b2a5-4105-4e4c-891f-2000fec8732a)

Reply:
The only reason I used those values is because we use them all over the app (ex: the cameras menu in camera card)

Reply:
Regarding the alignment, can you check what is the difference on the file explorer card?

It should have have same issue I think...

Reply:
It uses the `collapsable-card` instead, and the button is aligned there:
![image](https://github.com/matmen/fluidd/assets/25269274/8706ff58-ebad-4b76-8410-19079ab00340)

Reply:
I've fixed it, the problem was the margin on the search box (`mx-2`versus `ml-1` like on the FileSystemToolbar.vue)

![image](https://github.com/matmen/fluidd/assets/85504/aa2c67f8-b930-4a50-b86b-d2b3a6b24605)